manual_correction.py

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env python
 #
-# Script to perform manual correction of spinal cord segmentation, gray matter segmentation, vertebral labeling, and
-# pontomedullary junction labeling.
+# Script to perform manual correction of spinal cord segmentation, gray matter segmentation, vertebral labeling,
+# pontomedullary junction labeling, and centerline.
```

Comment:
The readme also contains lesion segmentation and PMJ labelling, which are missing here. Maybe those should be added for uniformity!

Reply:
Good catch! Fixed in https://github.com/spinalcordtoolbox/manual-correction/pull/33/commits/b8c6efd559f4afd85924357c02b3625263d07e95